extra-files/decompiled/SSDT-XOSI.dsl

```diff
@@ -1,28 +1,11 @@
-/*
- * Intel ACPI Component Architecture
- * AML/ASL+ Disassembler version 20190509 (64-bit version)
- * Copyright (c) 2000 - 2019 Intel Corporation
- * 
- * Disassembling to symbolic ASL+ operators
- *
- * Disassembly of iASLYWTc6v.aml, Thu May 28 19:06:11 2020
- *
- * Original Table Header:
- *     Signature        "SSDT"
- *     Length           0x00000143 (323)
- *     Revision         0x02
- *     Checksum         0x6C
- *     OEM ID           "DRTNIA"
- *     OEM Table ID     "XOSI"
- *     OEM Revision     0x00001000 (4096)
- *     Compiler ID      "INTL"
- *     Compiler Version 0x20190509 (538510601)
- */
 DefinitionBlock ("", "SSDT", 2, "DRTNIA", "XOSI", 0x00001000)
 {
     Method (XOSI, 1, NotSerialized)
     {
-        Local0 = Package (0x11)
+        // Based off of https://docs.microsoft.com/en-us/windows-hardware/drivers/acpi/winacpi-osi
+        // Add OSes from the above list as needed, most only check up to Windows 2015
+        // but check what your DSDT looks for
+        Local0 = Package (0x13)
```

Comment:
I'd remove `0x13` so package size is dynamically calculated.
If someone edits their SSDT before compiling, it'll use the correct size :)